rsqueakvm/main.py

```diff
@@ -79,6 +81,9 @@ def _usage(argv):
                                synthetic high-prio Process.
             -u|--stop-ui     - Only with -m or -r. Try to stop UI-process at
                                startup. Can help benchmarking.
+            --run-file       - Run the .st file supplied as first argument. No
```

Comment:
A bit of a hackish cmdline argument to support squeak-style invocation `./rsqueak --run-file Image run.st` but side-stepping the entire image startup. This will not work with many files, but with some (like e.g. scripts that just start and then exit, or benchmarking scripts)